scripts/vcpkg_ports/axom/vcpkg.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "axom",
+  "version-string": "develop",
+  "homepage": "https://github.com/llnl/axom",
+  "description": "Host-config generator for Axom TPLs",
```

Comment:
It currently only lists the dependencies/features that we want to build and generates the host-config.
It does not build axom, so that's why wrote it that way.

This is similar to the "fake" packages that we used to use in our `spack` setup.